VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participant-waiting-room/private-consultation-participants/private-consultation-participant-display-name/private-consultation-participant-display-name.component.scss

```diff
@@ -0,0 +1,7 @@
+.white {
+  color: rgba(255, 255, 255, 0.9);
+}
+
+.margin-right {
```

Comment:
Avoid naming something just based on what the current style we want to implement is. Change this to something like .display-name so if we need to add any further styles we can do so in here rather than having to create multiple classes.

Reply:
@FarronF Updated as suggested